src/node/rpc/endpoint.h

```diff
@@ -68,32 +68,23 @@ namespace ccf
       ForwardingRequired forwarding_required = ForwardingRequired::Always;
       ExecuteOutsideConsensus execute_outside_consensus =
         ExecuteOutsideConsensus::Never;
-      bool require_client_signature = false;
-      bool require_client_identity = true;
-      bool require_jwt_authentication = false;
+      std::vector<std::string> authn_policies = {};
 
       nlohmann::json openapi;
       bool openapi_hidden = false;
 
       MSGPACK_DEFINE(
         forwarding_required,
         execute_outside_consensus,
-        require_client_signature,
-        require_client_identity,
-        require_jwt_authentication,
+        authn_policies,
         openapi,
         openapi_hidden);
     };
 
     DECLARE_JSON_TYPE_WITH_OPTIONAL_FIELDS(EndpointProperties);
     DECLARE_JSON_REQUIRED_FIELDS(
-      EndpointProperties,
-      forwarding_required,
-      execute_outside_consensus,
-      require_client_signature,
-      require_client_identity);
-    DECLARE_JSON_OPTIONAL_FIELDS(
-      EndpointProperties, require_jwt_authentication, openapi, openapi_hidden);
+      EndpointProperties, forwarding_required, authn_policies);
+    DECLARE_JSON_OPTIONAL_FIELDS(EndpointProperties, openapi, openapi_hidden);
```

Comment:
Why are these optional? I assume this was because we didn't want to break existing applications when these fields were added but we may want to set them as required now as the changes in this PR are breaking anyway?